_sass/frogtek-theme.scss

```diff
@@ -231,7 +232,28 @@ footer {
 
 .empowering {
   font-family: 'ShadowsIntoLight';
-  font-size: 3em;
+  font-size: 3.5vw;
+}
+
+@media screen and (min-width: 1200px) {
+  .empowering {
+    font-size: 55px;
+  }
+}
+
+@media all and (max-width: 599px) and (min-width: 50px) {
```

Comment:
@cati-risquez You should only use Bootstrap thresholds for media queries: `540px | 720px | 960px | 1140px`, since all the layouts are built on them. I would rather define:

`@media screen and (max-width: 540px) {`